q-pup/motor_module/equations.txt

```diff
@@ -0,0 +1,6 @@
+﻿"N"= 21


```

Comment:
Is there the 40:1 variant too?

Reply:
To be done in follow-up PR. :eyes: @wraftus